.github/workflows/examples.yml

```
@@ -16,13 +16,24 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: setup-python${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install
+    - name: Setup cache
+      uses: actions/cache@v2
+      env:
+        cache-name: daily-example
+      with:
+        path: /opt/**/site-packages
+        key: ${{ runner.os }}-${{ matrix.python-version }}-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}
```

Could be over-engineering but we'll probably sooner or later want to manually overwrite/delete caches. The following is one way to prepare for it.

```suggestion
        key: ${{ runner.os }}-${{ matrix.python-version }}-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}-v1
```